aldryn_events/tests/frontend/unit/test.cl.events.js

```diff
@@ -3,20 +3,50 @@
  * @copyright: http://www.divio.ch
  */
 
-'use strict';
+(function () {
```

Comment:
in general I don't like this in tests, but jshint will complain and it makes sense to be consistent everywhere